activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Updated Unicode version to 9.0.0
+
+    *Fumiaki MATSUSHIMA*
+
```

Comment:
Worth noting how this affects grapheme cluster boundaries.

Notably, this will change `string.mb_chars.grapheme_length` and `string.mb_chars.reverse`—perhaps fixing them to work with clusters like 👩‍👩‍👧‍👦!